docs/governance.md

```diff
@@ -0,0 +1,300 @@
+
+* **Projects**: A single repository in the Parca [GitHub organization][gh] is referred to as a project.
+
+* **The Parca project**: The sum of all activities performed under this governance, concerning one or more repositories
+  or the community.
+
+## Values
+
+The Parca developers and community are expected to follow the values defined in the [CNCF charter][charter], including
+the [CNCF Code of Conduct][coc].
+Furthermore, the Parca community strives for kindness, giving feedback effectively, and building a welcoming
+environment.
+The Parca developers generally decide by consensus and only resort to conflict resolution by a majority vote if
+consensus cannot be reached.
+
+## Projects
+
+Each project must have a `MAINTAINERS.md` file with at least one maintainer.
+Where a project has a release process, access and documentation should be such that more than one person can perform a
+release.
+Releases should be announced on the [parca-users mailing list][users].
+Any new projects should be first proposed on the [developers mailing list][devs] following the voting procedures listed
+below.
+When a project is no longer relevant, it should be moved to the parca-junkyard GitHub organization.
+
+## Decision making
+
+### Team members
+
+Team member status may be given to those who have made ongoing contributions to the Parca project for at
+least `3 months`.
+This is usually in the form of code improvements and/or notable work on documentation,
+but organizing events or user support could also be taken into account.
+
+New members may be proposed by any existing member by email to [parca-team][team].
+It is highly desirable to reach consensus about acceptance of a new member.
+However, the proposal is ultimately voted on by a formal [supermajority vote](#supermajority-vote).
+
+If the new member proposal is accepted, the proposed team member should be contacted privately via email to confirm or
+deny their acceptance of team membership.
+This email will also be CC'd to [parca-team][team] for record-keeping purposes.
+
+If they choose to accept, the [onboarding](#onboarding) procedure is followed.
+
+Team members may retire at any time by emailing [the team][team].
+
+Team members can be removed by [supermajority vote](#supermajority-vote) on [the team mailing list][team].
+For this vote, the member in question is not eligible to vote and does not count towards the quorum.
+Any removal vote can cover only one single person.
+
+Upon death of a member, they leave the team automatically.
+
+In case a member leaves, the [offboarding](#offboarding) procedure is applied.
+
+The current team members are:
+
+* Frederic Branczyk ([Polar Signals](https://polarsignals.com))
+* Thor Hansen ([Polar Signals](https://polarsignals.com))
+* Matthias Loibl ([Polar Signals](https://polarsignals.com))
+* Kemal Akkoyun ([Polar Signals](https://polarsignals.com))
+* Sumera Priyadarsini ([Polar Signals](https://polarsignals.com))
```

Comment:
Yomi is a team member and maintainer of frontend

Reply:
:see_no_evil: Sorry that I've missed it

Reply:
He wasn't on the parca-team that's why I've missed it. I'm adding him to both.